From b21e57e713b5248e0b429524d4c2244ae7592e22 Mon Sep 17 00:00:00 2001
From: Paul Arthur <paul.arthur@flowerysong.com>
Date: Wed, 14 Feb 2018 15:32:53 -0500
Subject: [PATCH 1/3] libucl: Correct length of replacement character escape
 sequence

Previously the final character was chopped off, emitting '\uFFF'
instead of '\uFFFD'.
---
 contrib/libucl/ucl_emitter_utils.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contrib/libucl/ucl_emitter_utils.c b/contrib/libucl/ucl_emitter_utils.c
index 6c46a8e49..a2a69e443 100644
--- a/contrib/libucl/ucl_emitter_utils.c
+++ b/contrib/libucl/ucl_emitter_utils.c
@@ -130,7 +130,7 @@ ucl_elt_string_write_json (const char *str, size_t size,
 				break;
 			default:
 				/* Emit unicode unknown character */
-				func->ucl_emitter_append_len ("\\uFFFD", 5, func->ud);
+				func->ucl_emitter_append_len ("\\uFFFD", 6, func->ud);
 				break;
 			}
 			len = 0;
-- 
2.14.3

